node/typings.ts

```diff
@@ -1,18 +1,18 @@
-import { ServiceContext } from '@vtex/api'
-
-import { Clients } from './clients'
+import { ServiceContext, RecorderState, ParamsContext, IOClients } from '@vtex/api'
 
 declare global {
-  type Context = ServiceContext<Clients, State>
+  type Context = ServiceContext<IOClients, State, Custom>
 
-  interface State extends BroadcasterEvent {
+  interface State extends RecorderState, BroadcasterEvent {
     alwaysNotify: boolean
   }
 
   interface BroadcasterEvent {
     HasStockKeepingUnitModified: boolean
     IdSku: string
   }
+  interface Custom extends ParamsContext {
```

Comment:
Why create a `Custom` empty interface instead of using `ParamsContext` directly?

Reply:
That is unnecessary, will change 👍